src/Sarif/NotYetAutoGenerated/ReportingDescriptorEqualityComparer.cs

```diff
@@ -1,339 +0,0 @@
-// Copyright (c) Microsoft.  All Rights Reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-using System;
-using System.CodeDom.Compiler;
-using System.Collections.Generic;
-
-using Microsoft.CodeAnalysis.Sarif.Readers;
-
-namespace Microsoft.CodeAnalysis.Sarif
-{
-    /// <summary>
-    /// Defines methods to support the comparison of objects of type ReportingDescriptor for equality.
-    /// </summary>
-    [GeneratedCode("Microsoft.Json.Schema.ToDotNet", "1.1.3.0")]
-    internal sealed class ReportingDescriptorEqualityComparer : IEqualityComparer<ReportingDescriptor>
```

Comment:
>ReportingDescriptorEqualityComparer [](http://example.com/codeflow?start=26&length=35)

This is removed because we fixed the issue for Dictionary type of data model (`MultiformatMessage` in this case) in EqualityComparer code generator, so the path version of this EqualityComparer is not required any more.